validator/sawtooth_validator/journal/chain.py

```diff
@@ -484,6 +484,24 @@ def run(self):
 
             # 4) Evaluate the 2 chains to see if the new chain should be
             # committed
+            LOGGER.info(
```

Comment:
This stuff should be on one logging line, or the multiple lines should make sense on their own.